plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2865,6 +2875,7 @@ action.ide.reload_code.dialog=<html>If you load file from file system, all chang
   you made to the code will be overwritten!<br>\
   <b>Are you sure you want to continue?
 common.confirmation=Confirmation
+common.mod_element_name=Mod element name
 common.name_already_exists=This name already exists
```

Comment:
This string is now only used in a single place, is it OKay if I use the new `validators.unique_name.duplicate` string in that place?

Reply:
This would reduce clarity for cost of one less string, I don't think it makes sense